src/components/Slider/index.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from "react";
+import LatestWorkshopSlideCard from "./LatestWorkshopSlideCard";
+import "./index.css";
+
+export default function LatestWorkshopSlider({ title, subTitle, slides }) {
+  const [current, setCurrent] = useState(0);
+
+  return (
+    <div className="slider-container">
+      <div className="slider-title-group">
+        <h2>{title}</h2>
+        <p>{subTitle}</p>
+      </div>
+      <div className="slider-slides">
+        <span
+          className="slide-control"
+          onClick={() => {
+            current !== 0
+              ? setCurrent(current - 1)
+              : setCurrent(slides.length - 1);
+          }}
+        >
+          &#8810;
+        </span>
+        <LatestWorkshopSlideCard slides={slides} current={current} />
```

Comment:
If this only uses one slide, you should store that slide in state OR just pass that slide like so: 
```suggestion
        <LatestWorkshopSlideCard slide={slides[current]} />
```

The way you currently access it, is forcing the `<LatestWorkshopSlideCard/>` component to find it in an array every time it wants to use a field such as title, image or location. By accessing the array once and then providing the specific object it can be a lot more performant 😊

Reply:
This is exactly where I was feeling weird when passing the currentIndex as a props to the WorkshopSlideCard, I updated as the way you suggested here with the renamed `currentSlideIndex`